_includes/static/navbar.html

```diff
@@ -4,7 +4,10 @@
     <div class="col">
       <div class="row justify-content-start">
         <div class="col-sm-4">
-          <img class="nav-img" src="{{ site.baseurl }}/assets/img/nav-img.svg">
+          <a href="https://www.sack.edu.lk/#gsc.tab=0">
```

Comment:
```suggestion
          <a href="../../">
```

@VishwaAmarasinghe We can directly use the relative path here.

Reply:
Ok brother I updated that

Reply:
I think it should be `<a href="{{ site.baseurl }}">`

Reply:
@VishwaAmarasinghe could you please the above suggestion

Reply:
Ok aiyya